compass/landice/tests/antarctic/antarctic.cfg

```diff
@@ -0,0 +1,29 @@
+# config options for antarctic test cases
+[antarctic]
+
+# number of levels in the mesh
+levels = 10
+
+# distance from ice margin to cull (km).
+# Set to a value <= 0 if you do not want
+# to cull based on distance from margin.
+cull_distance = 5.0
+
+# mesh density parameters
+# minimum cell spacing (meters)
+min_spac = 1.e4
+# maximum cell spacing (meters)
+max_spac = 1.e5
+# log10 of max speed (m/yr) for cell spacing
+high_log_speed = 2.5
+# log10 of min speed (m/yr) for cell spacing
+low_log_speed = 0.75
+# distance at which cell spacing = max_spac (meters)
+high_dist = 1.e5
+# distance within which cell spacing = min_spac (meters)
+low_dist = 1.e4
```

Comment:
Unless I missed something, these aren't being used (yet?).

Reply:
Those get used here: https://github.com/hollyhan/compass/blob/compass_add_AISmeshGen/compass/landice/mesh.py#L177-L179

Reply:
Thanks @trhille. In that case, the comments here should clarify what the distance is to or between.

Reply:
Also, the shared function should probably document, rather than just list, these config options. We don’t need to change that in this PR, though.

Reply:
@hollyhan, Yeah that's correct. But maybe say `distance to ice edge or grounding line`. Currently, we use `high_dist` and `low_dist` in both the `use_distance_to_grounding_line` and  `use_distance_to_ice_edge` cases, which should probably be changed eventually.